training/src/app/app.component.ts

```diff
@@ -6,5 +6,8 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'training';
+  title = 'chuck-norris';
 }
+
+// Search by key component (2) - should perform a get request (via service) with the user input - after the user clicks on the button OR 1 second after the user finished typing.
```

Comment:
always try to make sure u dont leave comments when pushing your code to the outside world